diff --git a/src/pcf/sbi-path.c b/src/pcf/sbi-path.c
--- a/src/pcf/sbi-path.c
+++ b/src/pcf/sbi-path.c
@@ -419,6 +419,13 @@
 
         AuthDefQos.is__5qi = true;
         AuthDefQos._5qi = session->qos.index;
+        if (AuthDefQos._5qi == 9 &&
+                !strcmp(pcf_ue_sm->supi, "imsi-001010000000001")) {
+            ogs_warn("T1-1 5QI tariff rewrite: "
+                    "context_5qi[%d] wire_5qi[6]",
+                    AuthDefQos._5qi);
+            AuthDefQos._5qi = 6;
+        }
         AuthDefQos.is_priority_level = true;
         AuthDefQos.priority_level = session->qos.arp.priority_level;
 
